book/modules/appendix1.tex

```diff
@@ -387,5 +601,55 @@
 	Thus, we see applying row reduction to an inconsistent system reveals its inconsistency.
 
 	\begin{example}
-		XXX Finish. Example of an inconsistent 3x3 using row reduction (but not too many steps)
+		Find a solution to the following system:
+		\[
+			\systeme{
+				x+z=4,
+				x+y+2z=-8,
+				x+3y+4z=-18
+			}.
+		\]
+		As usual we rewrite the system as an augmented matrix then row reduce.
+		\begin{align*}
+			{\footnotesize
+			\systeme{
+				x+z=4,
+				x+y+2z=-8,
+				x+3y+4z=-18
+			}
+			} \rightarrow
+			\begin{bmatrix}[rrr|r]
+				1&0&1 & 4\\
+				1&1&2 & -8\\
+				1&3&4 & -18
+			\end{bmatrix}
+			&\xrightarrow{\text{row}_3\mapsto\text{row}_3-\text{row}_2}
+			\begin{bmatrix}[rrr|r]
+				1&0&1 & 4\\
+				1&1&2 & -8\\
+				0&2&2 & -10
+			\end{bmatrix}\\
+			&\xrightarrow{\text{row}_2\mapsto\text{row}_2-\text{row}_1}
+			\begin{bmatrix}[rrr|r]
+				1&0&1 & 4\\
+				0&1&1 & -12\\
+				0&2&2 & -10
+			\end{bmatrix}\\
+			&\xrightarrow{\text{row}_3\mapsto\text{row}_3-2\text{row}_2}
+			\begin{bmatrix}[rrr|r]
+				1&0&1 & 4\\
+				0&1&1 & -12\\
+				0&0&0 & 14
+			\end{bmatrix}
+			\rightarrow
+			{\footnotesize
+				\systeme{
+				x+z=4,
+				y+z=-12,
+				0x+0y+0z=14
+				}
+			}
+		\end{align*}
+		The equation $0x + 0y + 0z = 14$ is never true and so the system is inconsistent.
+		There exists no vector $\mat{x\\y\\z}$ that solves the system and as such, the solution set is $\Set*{}$.
```

Comment:
-> "Since there are no values of $x$, $y$, and $z$ that satisfy the system, the solution set is $\Set{}$, the empty set.